dist/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf8">
+    <title>Sigsa.io</title>
+  </head>
+  <body>
+    <div id="reservation"></div>
+    <div id="menu"></div>
+    <div id="reviews"></div>
+    <footer>Contributors: Minho Choo, Jennie Zeng, Priyanka Pramanik </footer>
+
+    <script src="http://localhost:3002/bundle.js"></script>
+    <script src="http://localhost:3003/main.js"></script>
+    <script src="http://localhost:3004/bundle.js"></script>
+
+  </body>
+</html>
```

Comment:
need additional line at the end of the file

Reply:
extra line added in new commit